pkg/act/act.go

```diff
@@ -143,7 +143,6 @@ func NewACT(in struct {
 	in.Jobs.Add(job.OneShot("act-metrics-init",
 		func(ctx context.Context, health cell.Health) error { return a.update(ctx) },
 	))
-	in.Lifecycle.Append(in.Jobs)
```

Comment:
Just noting that the diff pattern for this file was different from others. Not sure if there's some magic in how `cell.Invoke()` registers the lifecycle for the jobs input parameter or if something needs changing here. Will defer to you.

Reply:
All Appends of job.Group were removed (it no longer implements HookInterface, instead NewGroup appends)

Reply:
I see. Nice cleanup but yeah not necessary for this particular PR. :+1: